Makefile

```diff
@@ -37,4 +43,7 @@ index:
 index_local:
 	sleep 10
 	python3 genomic_data_service/region_indexer.py --local
+index_docker:
+	sleep 10
+	DOCKER=index_docker python3 genomic_data_service/region_indexer.py --local
```

Comment:
If we make use of `argparse` we can get rid of the environment dependent configuration in `region_indexer`. See comment on `region_indexer.py`.